operator/pkg/translate/translate_value.go

```diff
@@ -271,6 +260,45 @@ func (t *ReverseTranslator) setEnablementFromValue(valueSpec map[string]interfac
 	return nil
 }
 
+// IssueWarningForGatewayK8SSettings creates deprecated warning messages
+// when user try to set kubernetes settings for gateways via values api.
+func (t *ReverseTranslator) IssueWarningForGatewayK8SSettings(valuesOverlay string) (string, error) {
```

Comment:
maybe drop Issue from the name? it suggests the function will print something.

Reply:
done